docs/docs/tutorials/dask-nebari-setup.md

```diff
@@ -0,0 +1,175 @@
+
+```python
+profiles:
+  jupyterlab:
+    - display_name: Small Instance
+      description: Stable environment with 1 cpu / 1 GB ram
+      access: all
+      default: true
+      kubespawner_override:
+        cpu_limit: 1
+        cpu_guarantee: 1
+        mem_limit: 1G
+        mem_guarantee: 1G
+    - display_name: Medium Instance
+      description: Stable environment with 1.5 cpu / 2 GB ram
+      access: yaml
+      groups:
+        - admin
+        - developers
+      users:
+        - bob
+      kubespawner_override:
+        cpu_limit: 1.5
+        cpu_guarantee: 1.25
+        mem_limit: 2G
+        mem_guarantee: 2G
+    - display_name: Large Instance
+      description: Stable environment with 2 cpu / 4 GB ram
+      access: keycloak
+      kubespawner_override:
+        cpu_limit: 2
+        cpu_guarantee: 2
+        mem_limit: 4G
+        mem_guarantee: 4G
+  dask_worker:
+    "Small Worker":
+      worker_cores_limit: 1
+      worker_cores: 1
+      worker_memory_limit: 1G
+      worker_memory: 1G
+    "Medium Worker":
+      worker_cores_limit: 1.5
+      worker_cores: 1.25
+      worker_memory_limit: 2G
+      worker_memory: 2G
+```
+
+</details>
+
+**Accessing cluster options**
+
+```python
+from dask_gateway import Gateway
+gateway = Gateway()
+
+options = gateway.cluster_options()
+options
+```
+
+Once the `ipywidget` shows up we can customise as per our requirement.
+
+![Cluster Options](/img/cluster_options.png)
+
+:::note
+It’s important that the environment used for your notebook matches the dask worker environment.
+:::
+
+**Creating a dask cluster**
+
+```python
+cluster = gateway.new_cluster(options)
+cluster
+```
+
+We have the option to choose between `manual` scaling and `adaptive` scaling, the benchmarking stats would be
+useful here to decide the right set of configuration.
+Starting with the lowest possible configuration could be another helpful strategy.
+
+![Creating a Cluster](/img/cluster_creation.png)
+
+<details>
+<summary> Click on the dashboard URL to view Dask UI: Sample info </summary>
+
+![Dask UI resource utilisation](/img/dask_UI.png)
+
+</details>
+
+**Calling dask client to get started**
+
+```python
+
+client = cluster.get_client()
+client
+```
+
+<details>
+<summary> Click to view sample output: Dask client </summary>
+
+![dask client](/img/dask_client.png)
+
+</details>
+
+**Fun part, let's code with dask**
+
+```python
+import dask.array as da
+x = da.random.random((10000, 10000), chunks=(1000, 1000))
+y = x + x.T
+z = y[::2, 5000:].mean(axis=1)
+z.compute()
+```
+
+<details>
+<summary> Click to view sample output: Dask compute </summary>
+
+```shell
+array([0.99628925, 0.99659686, 1.00412466, ..., 0.99887597, 1.00219302,
+       1.0027488 ])
+```
+
+</details>
+
+### Dask diagnostic UI
+
+![dask diagnostic UI](/img/dask_diagostic_UI.png)
+
+### Conclusion
+
+Kudos ✨, we now have a working dask cluster inside Nebari.
+Excellent to get started with those large datasets.
```

Comment:
Need to include shutting down the cluster. This is why I'd like to show how we can use a context manager to handle the cluster ([like this](https://gist.github.com/kcpevey/3a9a94ca741decd180c08570a721a9c0)).

Should we show both approaches or just include the context manager here?

Reply:
I think there is some benefit to showing the "manual" approach. Perhaps we could add a note or something towards the bottom that makes a recommendation to use a context manager. 
Something like: 
"Wrapping the dask-gateway in a context manager, like this, is a great way to avoid having to write a lot of boilerplate, with the added benefit that it will ensure the cluster is fully shutdown once it's finished being used."

Reply:
should I add a separate section for the manual approach using `context manager`? 
Or just the recommendation at the end with the `sample code` ?

Reply:
Currently, I have added the `recommendation` with the code snippet. Happy to make any additional tweaks

Reply:
In the same code snippet, could we include the same example from above but being used by this context manager? 

Something like:

```python
with dask_cluster(...) as cluster:
    import dask.array as da
    x = da.random.random((10000, 10000), chunks=(1000, 1000))
    y = x + x.T
    z = y[::2, 5000:].mean(axis=1)
    z.compute()
```